scripts/qa_crowbarsetup.sh

```diff
@@ -4525,6 +4525,10 @@ function onadmin_prepare_crowbar_upgrade
         # move nodes to upgrade mode
         safely crowbar_api_request POST $crowbar_api /installer/upgrade/prepare.json
     else
+        # first check the command can complete successfully
+        safely crowbarctl upgrade repocheck crowbar
+
+        # then parse the command output
         if crowbarctl upgrade repocheck crowbar --format plain | grep "missing" ; then
```

Comment:
you were right, `safely` only applies to the command before the pipe. The `if` applies to the output of the grep. Updating the PR.